### PR TITLE
feat: site redesign

### PR DESCRIPTION
**Pull request by felangel**

### CODE REVIEW

File: .astro/types.d.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
+declare module 'astro:content' {
```
Comment (eseidel) [inline .astro/types.d.ts:1]: Are we supposed to check in .astro?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,19 @@
+MIT License
```
Comment (eseidel) [inline LICENSE:1]: This is fine for now.  Eventually I need to decide if this is CC BY (like the handbook) or MIT + Apache (like our other repos) or fine being just MIT (which is different from our other repos).

File: src/assets/icons/TwitterIcon.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const TwitterIcon = () => (
```
Comment (eseidel) [inline src/assets/icons/TwitterIcon.jsx:1]: I'm amused that these are dynamic at all and not just static .svg files.

**Review by eseidel — APPROVED** [review]
Big step forward.  Thank you.

**Comment by eseidel** [comment]
Are we going to be sad at ourselves later for checking in movie files? 🤷

**Comment by eseidel** [comment]
re: movie files, does git lfs help at all?

File: .astro/types.d.ts (hunk 1 above)
Comment (felangel) [reply .astro/types.d.ts]: good catch

File: src/assets/icons/TwitterIcon.jsx (hunk 3 above)
Comment (felangel) [reply src/assets/icons/TwitterIcon.jsx:1]: They're components which can be embedded anywhere in pages/other components

**Comment by felangel** [comment]
> re: movie files, does git lfs help at all?

The movie file is ~99mb so not sure whether it warrants git lfs 🤷

**Pull request merged**